package.json

```diff
@@ -221,23 +225,25 @@
     "sinon": "^7.3.1",
     "sinon-chai": "^3.3.0",
     "stream-to-string": "^1.1.0",
-    "tmp": "^0.1.0"
+    "tmp": "^0.1.0",
+    "ts-node": "^8.5.4",
+    "typescript": "^3.7.3"
   },
   "scripts": {
     "build-browser-example": "BABEL_ENV=browser browserify example/index.js -o dist/browser-example.js -t babelify",
     "build-local-continuous": "babel src --watch -d lib --ignore '**/*_spec.js'",
-    "build-local": "babel src -d lib --ignore '**/*_spec.js'",
+    "build-local": "rm -r lib && tsc -p tsconfig.node.json && mv lib/src/* lib/ && rm -r lib/src lib/package.json",
     "build-release": "BABEL_ENV=browser browserify scripts/cucumber.js -o dist/cucumber.js -t babelify --standalone Cucumber --debug",
     "feature-test": "node ./bin/cucumber-js",
-    "lint-autofix": "eslint --fix \"{example,features,scripts,src,test}/**/*.js\"",
-    "lint-code": "eslint \"{example,features,scripts,src,test}/**/*.js\"",
+    "lint-autofix": "eslint --fix \"{example,features,scripts,src,test}/**/*.ts\"",
+    "lint-code": "eslint \"{example,features,scripts,src,test}/**/*.ts\"",
     "lint-dependencies": "dependency-lint",
     "lint": "yarn run lint-code && yarn run lint-dependencies",
     "prefeature-test": "yarn run build-local",
     "prepublishOnly": "rm -r lib && yarn run build-local",
     "test-coverage": "yarn run lint && ./scripts/test-coverage.sh",
     "test": "yarn run lint && yarn run unit-test && yarn run feature-test",
-    "unit-test": "mocha src"
+    "unit-test": "mocha 'src/**/*_spec.ts'"
```

Comment:
A bit petty but I think if we changed filenames to fit the common `.spec.ts` convention we would probably just need `mocha` here.

Reply:
Auhh interesting. Does mocha support typescript natively?

Reply:
Whoops nope, I was wrong. I figured if we added:

```yaml
extension:
  - ts
```

it would be enough to pick up `*.spec.ts` by default, but looks like the [default spec glob in Mocha](https://github.com/mochajs/mocha/blob/master/example/config/.mocharc.yml#L41) doesn't quite account for the extension(s) and is prescriptive about the path starting with `test/` anyway.

Reply:
(I think Jest and/or Ava are better in this regard)

Reply:
Okay, sticking with what we have for now